src/lib/_imports/settings/font.css

```
@@ -36,11 +36,13 @@ Styleguide Settings.CustomProperties.Font
   /* Weight */
   /* https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight */
   /* NOTE: Long names to be consistent (opinions welcome) */
+  --global-font-weight--light: 300;
   --global-font-weight--regular: 400;
   --global-font-weight--medium: 500;
   --global-font-weight--bold: 700;
   --global-font-weight--black: 800;
   /* NOTE: Short names to increase adoption (opinions welcome) */
+  --light: var(--global-font-weight--light);
   --regular: var(--global-font-weight--regular);
   --medium: var(--global-font-weight--medium);
   --bold: var(--global-font-weight--bold);
```

@R-Tomas-Gonzalez, I think I will change this (in a mini follow-up PR) to the following, so one need **NOT** track variable inheritance when debugging in dev-tools (see [video](https://github.com/TACC/Core-Styles/assets/62723358/6454210d-8031-4055-9bec-4901549804d2)). What do you think?

```suggestion
  --light: 300;
  --global-font-weight--light: 300;
  --regular: 400;
  --global-font-weight--regular: 400;
  --medium: 500;
  --global-font-weight--medium: 500;
  --bold: 700;
  --global-font-weight--bold: 700;
  --black: 800;
  --global-font-weight--black: 800;
```

---

I.e. Rather than look up `--light` only to find `--global-font-weight--light` which finally leads one to `300`, instead one can look up `--light` to see `300`.

**Pros** of current "extra step" is learning the inheritance and having standard name. **Cons** of current "extra step" is… it's an extra step, and I don't think the verbose `--global-font-weight--light` is necessary (I never use it).

---

Hmm... Well. My question is why do we have both in the first place? If they're all the same, we should just use the one word variable. I'd structure this file like this"
```
// variables could be used globally (optional comment – of your choice)

--light: 300;
--regular: 400; (so on and so forth)
```

Then the developer can assume, the variable is global and that it's used here for font-weight, or what ever they use it on:
```
font-weight: --light;
```

The file name can also be more verbose, like `font-weight-variables.css` or something like that. Then we FOR SURE know what these variables are used for.

---

Thank you.

The `--global-font-weight--…` existed only for consistency with other verbose variables. I dislike like the verbose names[^1] for fonts. **I will replace them with just short names.** If there are too many uses (I found some) across repos, I'll support verbose ones for backward-compatibility.

_I will not consider renaming the file yet. I don't have bandwidth to explore backwards-compatibility of file naming, and everything else that can bring up._

[^1]: I've been disliking the verbose names for all other vars more every year. I can trim them too, but will need to support them for backwards-compatibility until v3.

---

@wesleyboar, to do:
1. ```suggestion
     --light: 300;
     --regular: 400;
     --medium: 500;
     --bold: 700;
     --black: 800;
     
     /* backwards-compatibility, version 0—2 */
     --global-font-weight--light: var(--light);
     --global-font-weight--regular: var(--regular);
     --global-font-weight--black: var(--black);
     --global-font-weight--medium: var(--medium);
     --global-font-weight--bold: var(--bold);
    ```

2. (At least in Core-Styles) Replace existing use of verbsoe name.
3. (Optional) Review existing use of verbose name in all client repos.

---

^

---

Task 1 is performed in:
- #306

 That task is performed in isolation to avoid testing, because it seems so simple and safe.